kotlinx-coroutines-core/jvm/test/VirtualTimeSource.kt

```diff
@@ -20,40 +24,38 @@ internal inline fun withVirtualTimeSource(log: PrintStream? = null, block: () ->
     }
 }
 
-private const val NOT_PARKED = -1L
-
 private class ThreadStatus {
     @Volatile @JvmField
-    var parkedTill = NOT_PARKED
+    var parkedTill: ComparableTimeMark? = null
     @Volatile @JvmField
     var permit = false
     var registered = 0
-    override fun toString(): String = "parkedTill = ${TimeUnit.NANOSECONDS.toMillis(parkedTill)} ms, permit = $permit"
+    override fun toString(): String = "parkedTill = $parkedTill, permit = $permit"
```

Comment:
Could you please tell how important is it to present the `parkedTill` value in milliseconds ?
It will require storing the `VirtualTimeSource.zero` in `ThreadStatus`.

Reply:
Reverted